src/Motion.js

```diff
@@ -42,22 +49,22 @@ const Motion = createReactClass({
       lastIdealStyle: currentStyle,
       lastIdealVelocity: currentVelocity,
     };
-  },
+  }
 
-  wasAnimating: false,
-  animationID: (null: ?number),
-  prevTime: 0,
-  accumulatedTime: 0,
+  wasAnimating: false;
```

Comment:
This is why flowtype fails. `wasAnimating` and others are prototype properties, not instance properties. So as soon as you put them all to constructor like `this.wasAnimating = false`, tests will pass (I think so)